src/InstallationValidator.Core/Assets/Captions.cs

```diff
@@ -28,13 +28,12 @@ public static string ValidationDescription
             sb.AppendLine("  - overall validation result (<b>Valid/ Invalid</b>)");
             sb.AppendLine("  - validation result for every test simulation (<b>Valid/ Invalid</b>)");
             sb.AppendLine("  - summary of the deviations (if any) for each <b>Invalid</b> simulation (numerical values and comparison plots)");
```

Comment:
I think we should also remove _(if any)_. 
Invalid simulation always has some deviations